bit/transaction.py

```diff
@@ -94,12 +104,13 @@ def sanitize_tx_data(unspents, outputs, fee, leftover, combine=True, message=Non
             messages.append((message, 0))
 
     # Include return address in fee estimate.
-    fee = estimate_tx_fee(len(unspents), len(outputs) + len(messages) + 1, fee, compressed)
```

Comment:
Can you please cache stuff here:

```
num_outputs = len(outputs) + len(messages) + 1
sum_outputs = sum(out[1] for out in outputs)
```

Reply:
Yeah, that is a lot better. Less code reuse.